### DISCUSSION TITLE
Check for updates before shutting down

### DISCUSSION THREAD

**Discussion by neoKushan** [mbround18/valheim-docker#126] [Ideas]
Hi,

Thank you kindly for your work on this, really great docker image and it has been working well for me. I do have a suggestion for the auto update script though - if there's no update available, don't shut down the server. This would allow for more frequent update checks as well, instead of just once per day (For those that would rather be updated asap).

I believe this is possible via SteamCMD, though it's a bit of a hacky workaround (Skip to the last two posts in this thread): https://steamcommunity.com/app/346110/discussions/0/535152511358957700/

If I were more familiar with rust I'd offer up a PR but alas it's all a bit greek to me.

**Comment by CosmicHorrorDev** [comment]
That sounds like a nice improvement to me. I can work on drafting it up as a `odin update` subcommand and submit a draft PR sometime over the next couple days.

**Reply by mbround18** [reply]
I tossed up a feature issue for this :) https://github.com/mbround18/valheim-docker/issues/129

**Reply by neoKushan** [reply]
Thank you!
